django_server/documents/FEATURES.md

```
@@ -0,0 +1,129 @@
+- Support for multiple file formats:
+  - Text: .txt, .md, .rtf
+  - Documents: .pdf, .doc, .docx, .odt
+  - Data: .csv, .json, .xml
+  - Spreadsheets: .xls, .xlsx
+  - Web: .html
+
+### Vector Integration
+- Document chunking for embeddings
+- Vector ID mapping for retrieval
+- Integration with multiple vector databases:
+  - Pinecone
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)